master/getting-started/kubernetes/installation/hosted/k8s-backend/calico.yaml

```diff
@@ -88,8 +88,8 @@ spec:
             # The Calico IPv4 pool to use.  This should match `--cluster-cidr`
             - name: CALICO_IPV4POOL_CIDR
               value: "10.244.0.0/16"
-            # Set the hostname based on the k8s node name.
-            - name: HOSTNAME
+            # Set based on the k8s node name.
+            - name: NODENAME
```

Comment:
I'm trying to minimize changes to v2.0 so that existing users don't have any surprises. 

Essentially, the only changes I want to port back to v2.0 are the ones required to make it keep working.